main/getting-started/before-using-agoric.md

```diff
@@ -15,42 +16,53 @@ git clone https://github.com/Agoric/agoric-sdk
 cd agoric-sdk
 yarn install
 yarn build
-yarn link-cli ~/bin/agoric
-agoric --version
 ```
 
-Then proceed to [starting a project](/getting-started/start-a-project.md).
+### Add the `agoric` command to your PATH
+
+The `agoric` command is a shell script that uses `yarn` to run the Agoric SDK from the `agoric-sdk` directory. To use it from any directory on MacOS, add it to your path:
+
+```shell
+yarn link-cli ~/bin/agoric # creates an Agoric directory to reference
+echo $PATH # show current PATH
+export PATH=$PATH:/Users/*USERNAME*/bin # adds the parent directory to your PATH
+echo $PATH # show updated PATH to include the parent directory of Agoric folder
+# RESTART YOUR SHELL FOR CHANGES TO TAKE EFFECT #
```

Comment:
Restarting your shell will make these changes disappear, I think.

Reply:
you're right one command adds it to the shell and then it'll disappear on restart... adding it permanently to your shell by modifying the ".bash_profile" file requires a restart.

Reply:
right... discussion of adding it permanently should go in the expanded section on PATH handling... which... btw... where did it go?

Reply:
sorry i removed the expanded section on PATH handling based on the comment that it was [somewhat awkward](https://github.com/Agoric/documentation/pull/714/files/c2282953f685c7cba759ae42bb221d0057a16a46#r965208761) so I consolidated and married them... you'd like me to revive that section and place it near the bottom with the expanded "Install `agoric` CLI" instructions?

Reply:
That was what i was suggesting, yes; I'm not sure it's a good suggestion, though. I'm juggling too much to give this the attention it deserves just now.

Any thoughts, @kennyrowe?

Reply:
Any opinions? I've got a video demonstrating [install here](https://youtu.be/Yp76WxbaNhQ).  [@kennyrowe ](https://github.com/kennyrowe) @turadg @dckc

Reply:
This is to replace the existing video explainer? I don't see a change to that effect in the PR.
The existing video has voice-over; I'd rather not lose that.

Reply:
i haven't slotted in the video... it's not deployed on Agoric channels yet that sample is on my personal YouTube set to private unless you have the link.

Reply:
@dckc I've amended the instructions - think we could resolve this conversation, push these changes across, and then I can create an up to date video with voiceover reflecting these awesome changes we've made? =)